static/app/views/performance/transactionSummary/transactionSpans/spanDetails/exclusiveTimeHistogram.tsx

```diff
@@ -0,0 +1,240 @@
+import {HistogramData} from 'sentry/utils/performance/histogram/types';
+import {
+  computeBuckets,
+  formatHistogramData,
+} from 'sentry/utils/performance/histogram/utils';
+import {SpanSlug} from 'sentry/utils/performance/suspectSpans/types';
+
+const NUM_BUCKETS = 50;
+const PRECISION = 0;
+
+type Props = WithRouterProps & {
+  eventView: EventView;
+  location: Location;
+  organization: Organization;
+  spanSlug: SpanSlug;
+};
+
+export default function ExclusiveTimeTimeSeries(props: Props) {
+  const {location, router, organization, eventView, spanSlug} = props;
+
+  const theme = useTheme();
+
+  const period = eventView.statsPeriod;
+  const start = eventView.start ? getUtcToLocalDateObject(eventView.start) : null;
+  const end = eventView.end ? getUtcToLocalDateObject(eventView.end) : null;
+  const {utc} = normalizeDateTimeParams(location.query);
+
+  const yAxis = ['spans_exclusive_time'];
+
+  return (
+    <Fragment>
+      <HeaderTitleLegend>
+        {t('Self Time Distribution')}
+        <QuestionTooltip
+          size="sm"
+          position="top"
+          title={t(
+            'Distribution buckets counts of the same self time duration for the selected span op and group.'
+          )}
+        />
+      </HeaderTitleLegend>
+      <ChartZoom
+        router={router}
+        period={period}
+        start={start}
+        end={end}
+        utc={utc === 'true'}
+      >
+        {zoomRenderProps => (
+          <SpanHistogramQuery
+            location={location}
+            orgSlug={organization.slug}
+            eventView={eventView}
+            numBuckets={NUM_BUCKETS}
+            precision={PRECISION}
+            span={spanSlug}
+            dataFilter="exclude_outliers"
+          >
+            {({histogram, isLoading, error}) => {
+              if (error) {
+                return (
+                  <ErrorPanel data-test-id="histogram-error-panel">
+                    <IconWarning color="gray300" size="lg" />
+                  </ErrorPanel>
+                );
+              }
+
+              const chartOptions = {
+                grid: {
+                  left: '10px',
+                  right: '10px',
+                  top: '40px',
+                  bottom: '0px',
+                },
+                colors: theme.charts.getColorPalette(yAxis.length - 2),
+                seriesOptions: {
+                  showSymbol: false,
+                },
+                tooltip: {
+                  trigger: 'axis' as const,
+                  // p50() coerces the axis to be time based
+                  valueFormatter: (value, _seriesName) =>
+                    tooltipFormatter(value, 'p50()'),
+                },
+                yAxis: {
+                  axisLabel: {
+                    color: theme.chartLabel,
+                    // p50() coerces the axis to be time based
+                    formatter: (value: number) => axisLabelFormatter(value, 'p50()'),
```

Comment:
You might want to change the coercion here to something count-like (check if your y-axis is showing as time instead of counts).